[2024-10-15 21:27:49,119] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados manual__2024-10-16T00:27:44.319544+00:00 [queued]>
[2024-10-15 21:27:49,124] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados manual__2024-10-16T00:27:44.319544+00:00 [queued]>
[2024-10-15 21:27:49,124] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-10-15 21:27:49,124] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-10-15 21:27:49,124] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-10-15 21:27:49,138] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2024-10-16 00:27:44.319544+00:00
[2024-10-15 21:27:49,143] {standard_task_runner.py:52} INFO - Started process 10685 to run task
[2024-10-15 21:27:49,146] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'manual__2024-10-16T00:27:44.319544+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpl18pu5em', '--error-file', '/tmp/tmpx8ggzhej']
[2024-10-15 21:27:49,147] {standard_task_runner.py:80} INFO - Job 39: Subtask extrai_dados
[2024-10-15 21:27:49,184] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados manual__2024-10-16T00:27:44.319544+00:00 [running]> on host luan-capella
[2024-10-15 21:27:49,231] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2024-10-16T00:27:44.319544+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-16T00:27:44.319544+00:00
[2024-10-15 21:27:50,187] {python.py:173} INFO - Done. Returned value was: None
[2024-10-15 21:27:50,195] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20241016T002744, start_date=20241016T002749, end_date=20241016T002750
[2024-10-15 21:27:50,241] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-10-15 21:27:50,262] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
